posthog/api/capture.py

```diff
@@ -44,6 +44,13 @@
 SESSION_RECORDING_EVENT_NAMES = ("$snapshot", "$performance_event")
 
 
+EVENTS_DROPPED_OVER_QUOTA_COUNTER = Counter(
+    "capture_events_dropped_over_quota",
+    "Events dropped by capture due to quota-limiting, per resource, team and token.",
+    labelnames=["resource", "token", "team"],
```

Comment:
```suggestion
    labelnames=["resource_type", "token", "team_id"],
```

We'll need to standardize on a common set of labels, I'd prefer to use `team_id` instead of `team`.
The `resource` -> `resource_type` is more prometheus-y, but optional.

Reply:
Fair enough - updated 👍